.github/workflows/release.yml

```diff
@@ -0,0 +1,61 @@
+name: Create Release PRs
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
```

Comment:
We need a filter on this to only run when the client directory changes, I think?

Reply:
That is now configured in `release-please.config.json`. Every time we merge release-please will run and check if any committed changes affect those paths and update the release pr as needed.